packages/office-ui-fabric-react/src/components/Image/Image.test.tsx

```diff
@@ -15,126 +14,109 @@ const testImage1x1 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYA
+      <div>
+        <Image
+          src={ testImage1x1 }
+          width={ 1 }
+          height={ 3 }
+          imageFit={ ImageFit.cover }
+          className='is-portraitFrame'
+        />
+      </div>
     );
+
+    component.find('img').simulate('load');
+    expect(component.find('.ms-Image-image--landscape')).toHaveLength(1);
   });
 
-  it('can cover a landscape (wide) frame with a square image', (done) => {
-    let root = document.createElement('div');
-    let onLoadingStateChange = (loadState: ImageLoadState) => {
-      if (loadState === ImageLoadState.loaded) {
-        let image = document.querySelector('.ms-Image.is-landscapeFrame .ms-Image-image') as Element;
-        try {
-          expect(image.className).to.contain('ms-Image-image--portrait');
-        } catch (e) { done(e); }
-        done();
-      }
-    };
-    document.body.appendChild(root);
-    ReactDOM.render<HTMLDivElement>(
-      <Image
-        src={ testImage1x1 }
-        width={ 3 }
-        height={ 1 }
-        imageFit={ ImageFit.cover }
-        className='is-landscapeFrame'
-        onLoadingStateChange={ onLoadingStateChange }
-      />, root
+  it('can cover a landscape (wide) frame with a square image', () => {
+    let component = mount(
+      <div>
+        <Image
+          src={ testImage1x1 }
+          width={ 3 }
+          height={ 1 }
+          imageFit={ ImageFit.cover }
+          className='is-landscapeFrame'
+
+        />
+      </div>
     );
+    component.find('img').simulate('load');
+    expect(component.find('.ms-Image-image--portrait')).toHaveLength(1);
   });
 
-  it('can cover a landscape (wide) parent element with a square image', (done) => {
-    let root = document.createElement('div');
-    let onLoadingStateChange = (loadState: ImageLoadState) => {
-      if (loadState === ImageLoadState.loaded) {
-        let image = document.querySelector('.ms-Image.is-frameMaximizedPortrait .ms-Image-image') as Element;
-        try {
-          expect(image.className).to.contain('ms-Image-image--portrait');
-        } catch (e) { done(e); }
-        done();
-      }
-    };
-    document.body.appendChild(root);
-    ReactDOM.render<HTMLDivElement>(
-      <div style={ { width: '20px', height: '10px' } }>
+  it('can cover a landscape (wide) parent element with a square image', () => {
+    let component = mount(
+      <div style={ { width: '10px', height: '20px' } }>
         <Image
           className='is-frameMaximizedPortrait'
           imageFit={ ImageFit.cover }
           maximizeFrame
           src={ testImage1x1 }
-          onLoadingStateChange={ onLoadingStateChange }
         />
-      </div>, root
-    );
+      </div>);
+
+    // Manually set client height and width since there is no DOM
+    Object.defineProperty(HTMLDivElement.prototype, 'clientHeight', { get: () => 10, configurable: true });
+    Object.defineProperty(HTMLDivElement.prototype, 'clientWidth', { get: () => 20, configurable: true });
```

Comment:
Manually set `clientHeight` and `clientWidth` because they're undefined without dom